web/ui/src/common/miscUtils.js

```diff
@@ -207,6 +207,14 @@
                 return childCount;
             },
 
+            validateHostName: function(hostStr){
```

Comment:
Let's allow restricted ports for now since I'm not sure if we cases that need to override them or not.